src/rulesets/LinearAlgebra/dense.jl

```diff
@@ -394,3 +394,83 @@ function rrule(
     end
     return Ω, lyap_pullback
 end
+
+#####
+##### `kron`
+#####
+
+function frule((_, Δx, Δy), ::typeof(kron), x, y)
```

Comment:
Another point: Maybe it would be safer to restrict the types of the arguments to e.g.
```suggestion
function frule((_, Δx, Δy), ::typeof(kron), x::AbstractVecOrMat{<:Number}, y::AbstractVecOrMat{<:Number})
```
or possibly even
```suggestion
function frule((_, Δx, Δy), ::typeof(kron), x::AbstractVecOrMat{<:Real}, y::AbstractVecOrMat{<:Real})
```
(same applies to the other rules below)